### PR TITLE
Remove usage of deprecated JArrayHelper - com_modules

### PR DESCRIPTION
**Pull request by izharaazmi**
Remove usage of deprecated JArrayHelper from com_modules
#### Testing Instructions
- Test module manager overall including duplicate/clone modules.
- Review code.

### CODE REVIEW

File: administrator/components/com_modules/helpers/modules.php [hunk 1]
```diff
@@ -208,7 +210,7 @@ public static function getModules($clientId)
 			$modules[$i]->text = JText::_($module->text);
 		}
 
-		JArrayHelper::sortObjects($modules, 'text', 1, true, true);
+		ArrayHelper::sortObjects($modules, 'text', 1, true, true);
```
Comment (andrepereiradasilva) [inline administrator/components/com_modules/helpers/modules.php]: shouldn't this be?
```php
$modules = ArrayHelper::sortObjects($modules, 'text', 1, true, true);
```

**Comment by izharaazmi** [comment]
@andrepereiradasilva Thanks! Fixed it.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 74b583d7a1f18f15596d644b2008ff6ae674c6c4<br /><br />Made several copy, duplciate, publishe, unpublish taks in modules and all worked fine.
Code review seems fine.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10676">issues.joomla.org/joomla-cms/10676</a>.</sub>

**Comment by bertmert** [comment]
I have tested this item :white_check_mark: successfully on 74b583d7a1f18f15596d644b2008ff6ae674c6c4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10676">issues.joomla.org/joomla-cms/10676</a>.</sub>

**Comment by brianteeman** [comment]
RTC <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10676">issues.joomla.org/joomla-cms/10676</a>.</sub>

**Comment by izharaazmi** [comment]
**Milestone 3.7.0 ?**
Does this too count as a new feature? I must be missing some definition really. Can you help @wilsonge?

**Comment by wilsonge** [comment]
I'm closing this not because there's anything wrong but there's now a more comprehensive version for the entire admin section in https://github.com/joomla/joomla-cms/pull/10923 which is also RTC

**Pull request closed without merging**